components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx

```diff
@@ -285,4 +285,20 @@ class EIAnalyticsHorizontalBarChart extends Widget {
     }
 }
 
+function getCurrentPage() {
```

Comment:
Why these functions resides outside of the class?

Reply:
Fixed at b233d5335fc34cc9a03e24a4944cd39d0f6c3677